cabal.project

```diff
@@ -6,5 +6,5 @@ packages: prettyprinter
         , prettyprinter-compat-ansi-wl-pprint
         , prettyprinter-convert-ansi-wl-pprint
         , prettyprinter-compat-annotated-wl-pprint
-tests: true
-benchmarks: true
+-- tests: true
+-- benchmarks: true
```

Comment:
I’m unfamiliar with the cabal.project file, so I’ll just trust you on this one :-)

Reply:
recent `tasty` or `criterion` don't have install-plans on GHC-7.0 and GHC-7.2; unfortunately `cabal.project` doesn't support conditionals; so we cannot write

```haskell
if impl(ghc >= 7.4)
  tests: true
  benchmarks: true
```

to make tests and benchmarks build by default with recent GHC.

Not having benchmarks on older GHCs is completely fine. Tests are trickier, but I'm optimistic that we can extrapolate from other GHC results that "if it compiles, it works". 

One could miss some instance or symbol, and we do miss `Generic` stuff for GHC-7.0, so I'd recommend to try if `optparse-applicative` patch works on GHC-7.0; so those instances aren't essential for common use